index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Intuit Accounts - Sign Up</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body class="body">
+
+    <header class="sign-up">
+        <div class="sign-up-description">
+            <p class="sign-up-text">
+                Already have an Intuit account?
+            </p>
+            <a href="#" class="sign-up-link">
+                I forgot my User ID or Password
+            </a>
+        </div>
+
+        <button
+            class="sign-in-button"
+        >
+            Sign In
+        </button>
+    </header>
+
+    <div class="main-content">
```

Comment:
#### `HINT` 

:bulb: Using `main` here instead shows the dominant content of the `<body>` of a document. 
```suggestion
    <main class="main-content">
```

More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2: